tsconfig.json

```diff
@@ -7,7 +7,7 @@
         ],
         "module": "commonjs",
         "moduleResolution": "node",
-        "outDir": "out",
+        "outDir": "dist",
```

Comment:
When compiling tests, put in same folder. When when running tests that launch VSCode, it will not find the entrypoint defined `"main": "./dist/src/extension",`.
Thus, ensure all compile into same place.